src/consumer/mod.rs

```diff
@@ -266,6 +272,9 @@ where
 
     /// Commit the provided message. Note that this will also automatically
     /// commit every message with lower offset within the same partition.
+    /// This function is exactly equivalent to invoking [`Consumer::commit`]
+    /// with a [`TopicPartitionList`] which copies the topic and partition
+    /// from the message and adds 1 to the offset of the message.
```

Comment:
The evidence for this claim is right here: https://github.com/edenhill/librdkafka/blob/52949110d60e36c9eae59d31956a7fd760d783fa/src/rdkafka_offset.c#L428